components/cam/bld/configure

```diff
@@ -1028,9 +1000,6 @@ if ($phys_pkg =~ m/ideal|adiabatic|cam3|cam4/) {
 if ($clubb_sgs == 1) {
   $pbl_pkg = 'clubb_sgs';
 }
-if ($shoc_sgs == 1){
```

Comment:
I think we should go through all of the changes in this file and retain anything pertaining to shoc and p3

Reply:
agree